src/common/components/repositories-home/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { fetchUserRepositories } from '../../actions/repositories';
+
```

Comment:
Sure, I kind of wanted actions to be separate from components, but I guess in most cases it may mean new line under every import. So yep - let's group them all together.